node/subsystem-types/src/messages.rs

```diff
@@ -328,18 +328,13 @@ pub enum NetworkBridgeRxMessage {
 	NewGossipTopology {
```

Comment:
This struct no longer defines a topology, but a shulffling of discovery keys from which a topology is derived. Maybe that warrants a new name.